Framework/PythonInterface/mantid/api/src/Exports/DetectorInfo.cpp

```diff
@@ -0,0 +1,14 @@
+#include "MantidAPI/DetectorInfo.h"
+#include <boost/python/class.hpp>
+
+using Mantid::API::DetectorInfo;
+using namespace boost::python;
+
+void export_DetectorInfo() {
+  class_<DetectorInfo, boost::noncopyable>("DetectorInfo", no_init)
+      .def("size", &DetectorInfo::size, (arg("self")),
```

Comment:
@martyngigg The latter. I am considering supporting things like this:

```cpp
for (const auto &det : ws.detectorInfo()) {
  if (!det.isMasked())
    pos += det.position();
  // ...
}
```
where the type of `det` is some kind of iterator "proxy" forwarding all methods to `DetectorInfo`. On the other hand not much of our code is using such loops, since often we need an index to access other containers at the same time (or use our OpenMP macros that do not support range-based for), so maybe this is not an urgent addition.

Any thoughts?